examples/apps/cli/main.c

```diff
@@ -51,7 +51,7 @@ void __gcov_flush();
 #define OPENTHREAD_ENABLE_COVERAGE 0
 #endif
 
-#if OPENTHREAD_CONFIG_MULTIPLE_INSTANCE_ENABLE
+#if OPENTHREAD_CONFIG_EXTERNAL_HEAP_ENABLE
```

Comment:
```suggestion
#if OPENTHREAD_CONFIG_EXTERNAL_HEAP_ENABLE || OPENTHREAD_CONFIG_MULTIPLE_INSTANCE_ENABLE
```

Reply:
Actually, the platform APIs are required for only `EXTERNAL_HEAP_ENABLE` and we don't care `MULTIPLE_INSTANCE_ENABLE`. That says, we allow both internal and external heap for multiple OT instances.